<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gas_design_p2_1_dlg</class>
 <widget class="QWidget" name="gas_design_p2_1_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>底板岩巷大面积穿层钻孔抽采工作面瓦斯</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../usecad.qrc">
    <normaloff>:/images/cbm.ico</normaloff>:/images/cbm.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;煤层参数</string>
   </property>
   <widget class="QLineEdit" name="dip_angle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>煤层倾角</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thick">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;m&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>12</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;°&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>煤层厚度</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="head_graph">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>走 向 剖 面 图</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;回采巷道控制范围</string>
   </property>
   <widget class="QLineEdit" name="bottom">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>下帮</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="top">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>上帮</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="right">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>左帮</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>右帮</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="left">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>参考取值</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保 存</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;钻孔参数</string>
   </property>
   <widget class="QLineEdit" name="pore_gap">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>钻孔间距</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Dp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>钻孔直径</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;底板巷参数</string>
   </property>
   <widget class="QLineEdit" name="Hd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>底板巷高度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Wd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>底板巷宽度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="H_offset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>底板巷与工作面垂距</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>底板巷与工作面风巷水平投影距离</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="V_offset">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>两条底板巷的水平距离</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P_offset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>底板巷与工作面切眼水平投影距离</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="D_offset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="plane_graph">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>平 面 图</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;钻场参数</string>
   </property>
   <widget class="QLineEdit" name="Ws">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>钻场宽度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ls">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>钻场长度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="site_gap">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>钻场高度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>钻场间距</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Hs">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="dip_graph">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>倾 向 剖 面 图</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>&gt;&gt;工作面参数</string>
   </property>
   <widget class="QLineEdit" name="L2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>工作面长度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="L1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>顺槽长度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="h">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>巷道宽度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>巷道高度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="w">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>thick</tabstop>
  <tabstop>dip_angle</tabstop>
  <tabstop>Dp</tabstop>
  <tabstop>pore_gap</tabstop>
  <tabstop>L1</tabstop>
  <tabstop>L2</tabstop>
  <tabstop>w</tabstop>
  <tabstop>h</tabstop>
  <tabstop>Wd</tabstop>
  <tabstop>Hd</tabstop>
  <tabstop>V_offset</tabstop>
  <tabstop>H_offset</tabstop>
  <tabstop>D_offset</tabstop>
  <tabstop>P_offset</tabstop>
  <tabstop>top</tabstop>
  <tabstop>bottom</tabstop>
  <tabstop>left</tabstop>
  <tabstop>right</tabstop>
  <tabstop>help</tabstop>
  <tabstop>Ls</tabstop>
  <tabstop>Ws</tabstop>
  <tabstop>Hs</tabstop>
  <tabstop>site_gap</tabstop>
  <tabstop>save</tabstop>
  <tabstop>plane_graph</tabstop>
  <tabstop>head_graph</tabstop>
  <tabstop>dip_graph</tabstop>
 </tabstops>
 <resources>
  <include location="../usecad.qrc"/>
 </resources>
 <connections/>
</ui>
